### DISCUSSION TITLE
【灌水】是我不懂现代艺术？

### DISCUSSION THREAD

**Discussion by FeiXiao-zm** [share121/inter-knot#946] [其他]
最近去看展。旧都陷落时，芭莱大厦被空洞侵蚀的程度还没有现在这么深。空洞调查员紧急抢救出一批展品。
看完只想说，调查员冒着生命危险抢救这种东西，真的有必要吗？
《抱邦布的男人（们）》《以骸背上的芭莱双子》是我最不喜欢的作品，感觉只是戏仿名作，顺便拍下芭莱老板的马屁，作者是金斯顿．伯德，据说这个人没有经过学院派的训练，只是一个临摹名画的画工，这都可以吗？
![Screenshot_20240810_193110](https://github.com/user-attachments/assets/a5293d37-852a-40e3-913b-c8ed39fab067)

**Comment by warning-Kirov-reporting** [comment]
我觉得我上我也行

**Comment by gaojiaobei** [comment]
你冤枉金斯顿·伯德啦哈哈哈。

他没有接受过系统的美术训练，青年时期在复制画工厂工作，每天临摹名画12个小时，上班上崩溃了，所以恶作剧了一下，在名画里加入了一些无厘头元素。

结果画作落入了芭莱兄弟手中，芭莱兄弟惊为天人，想要联系他。结果他对画廊经理说“买我画的人品昧也太差了吧”。

我觉得他很清楚自己画的画就是垃圾，不过能够狠狠地宰芭莱兄弟两个冤大头，就是爽啦哈哈哈

**Comment by Armor-oss** [comment]
芭莱兄弟收藏的并不都是垃圾，他们赞助过一个叫大卫·吉尔伯特的艺术家，我很喜欢。

大卫幼年时卷入空洞幸存的经历，影响了他的艺术风格。所以他的画作如《空洞中的舞女》《西诺略丽莎》里，常常出现巨大以骸，是他童年亲眼所见的。

**Comment by Mayf1ower** [comment]
大卫·吉尔伯特画作里描绘的以骸，并没有恐怖的感觉。总觉得他画以骸的时候充满了感情···可能是童年经历给他留下了特殊的情结？
我记得他好像私下说过“让我改画作细节的芭莱兄弟远比以骸可怕”哈哈哈哈哈。
